src/gleam/fetch/fetch_options.gleam

```diff
@@ -0,0 +1,184 @@
+import gleam/dynamic.{type Dynamic}
+
+/// Gleam equivalent of JavaScript [`RequestInit`](https://developer.mozilla.org/docs/Web/API/RequestInit).
```

Comment:
Wrap doc comments at 80 lines like regular Gleam code, and included documentation for each option rather solely linking to an external resource 🙏